api/api.proto

```diff
@@ -4043,6 +4043,27 @@ message Job {
     google.protobuf.Timestamp last_update_time = 6;
 }
 
+message Schedule {
+  // ID of the schedule
+  string id = 1;
+  // StartTime is the scheduled time to start a run
+  // Valid formats: 
+  // daily=HH:mm  e.g. daily=19:15
+  // weekly=weekday@HH:mm  e.g. weekly=Sunday@19:15
+  // monthly=dd@HH:mm  e.g. monthly=22@19:15
+  string start_time = 2;
+  // MaxDurationHours defines the time limit of one run in hours 
+  double max_duration_hours = 3;
+  // Type is the type of job to run
+  Job.Type type = 4;
```

Comment:
Is Job.Type not part of Job that is repeated below?

Reply:
If it is intended to be a Schedule Type lets use that

Reply:
Yes it's intended to denote the schedule type. And it should be one of the Job.Type.